### DISCUSSION TITLE
利用Python來更容易取得觀測數據

### DISCUSSION THREAD

**Discussion by henrywen547** [1101-GS4719-C/Python-Community-GS4719-C#585] [General]
以現在的技術來進行天文觀測，雖然相較於以往的肉眼觀測更精確許多，然而也造成大量數位資料無法被消化。

Python能幫忙更輕鬆分析資料、建立模型，簡單易讀的特性也受科學家青睞。

而最近有一群太陽物理學家在網路共同開發基於 Python 的**公開**函式庫，讓太陽觀測數據及資料更容易取得。

這樣的應用讓我看到了一個python的新應用，也使曾經的天文社社員的我眼睛為之一亮!

[更容易取得觀測數據，太陽物理學家以 Phython 基礎聯合打造 SunPy](https://technews.tw/2020/02/28/exploring-our-star-with-sunpy/)

**Comment by gnsJhenJie** [comment]
[SkyField](https://rhodesmill.org/skyfield/) 同樣是Python的天文相關函式庫，可以用於計算人造衛星與星體的軌道，像是官網有個範例是計算某顆衛星一天會飛過地球的某個位置幾次。

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，原來python有這種功能。

**Comment by HsuChen88** [comment]
Python的應用真的很廣泛~

**Comment by cordydeng** [comment]
原來python還能有這個用途，長知識了~

**Comment by Jay-722** [comment]
Python真的在很多地方都能利用，非常厲害

**Comment by lx3211** [comment]
python真的是很常見用來分析數據的語言

**Comment by d26488399** [comment]
感謝分享，PYTHON的用途真的是超乎想像

**Comment by Sazmafuji** [comment]
物理套用python，python真的是在任何地方可以用
